src/deadline_test_fixtures/deadline/worker.py

```diff
@@ -635,6 +652,13 @@ class WindowsInstanceBuildWorker(WindowsInstanceWorkerBase):
 
     def configure_worker_command(self, *, config: DeadlineWorkerConfiguration) -> str:
         """Get the command to configure the Worker. This must be run as Administrator."""
+
+        password = None
+        if config.windows_user_secret:
+            user_secret = self.get_windows_user_secret(secret_id=config.windows_user_secret)
```

Comment:
Right now, the test runner is the one fetching the value of the secret, but then putting the secret in the contents of the SSM run command.

To avoid sending the password in the SSM command, we could instead have the worker fetch the secret value and construct the argument to `--password`. It would require granting the instance profile permissions to retrieve the secret value (`secretsmanager:GetSecretValue`).

What do you think?